layouts/shortcodes/blocks/case-studies.html

```diff
@@ -8,16 +8,24 @@
     <h3  style="text-align: center"><a href="/case-studies/" style="color: #3371E3; font-weight: 400">{{ $caseStudiesSection.LinkTitle }}</a></h3>
     <div id="caseStudiesWrapper">
       {{ range $caseStudiesPages }}
-			{{ $logo := .Resources.GetMatch "**{feature,logo}*.svg" }}
+			{{ $dark := .Resources.GetMatch "**{dark_logo}*.svg" }}
+			{{ $logo := .Resources.GetMatch "**{featured_logo}*.svg" }}
+			
```

Comment:
I'd like to treat a dark-mode preference and a light-mode preference as equally valid. (So, we should change this code to look up:
```
        {{ $logo_dark := .Resources.GetMatch "**{featured_dark_logo}*.svg" }}
        {{ $logo_light := .Resources.GetMatch "**{featured_light_logo}*.svg" }}
```
and maybe also look for `**{featured_common_logo}*.svg` (used for cases where one logo suits either a light or dark background).

Reply:
In that case i have do name change for all case studies items and accordingly update the logic. it will make little confuse/big in this PR. i want to do this change in another PR.

Reply:
I'm wary that the follow up work might not get done, and that we do end up with an implicit light-over-dark preference.

I suggest:
- ensure all the logos do work right on dark and light backgrounds
- let me know that's done
- I'll do this bit of the work and let you know the Git commands to update your fork to include that work.

Reply:
i will take responsibility of implementing this changes after this PR get merged.